commands/command.md

```diff
@@ -65,26 +93,25 @@ one argument, but up to an unlimited number: `MGET _key1_ [key2] [key3] ...`.
 Also note with `MGET`, the -1 value for "last key position" means the list
 of keys may have unlimited length.
 
-### Flags
+## flags
 Command flags is @array-reply containing one or more status replies:
 
-  - *write* - command may result in modifications
-  - *readonly* - command will never modify keys
-  - *denyoom* - reject command if currently out of memory
-  - *admin* - server admin command
-  - *pubsub* - pubsub-related command
-  - *noscript* - deny this command from scripts
-  - *random* - command has random results, dangerous for scripts
-  - *sort\_for\_script* - if called from script, sort output
-  - *loading* - allow command while database is loading
-  - *stale* - allow command while replica has stale data
-  - *skip_monitor* - do not show this command in MONITOR
-  - *asking* - cluster related - accept even if importing
-  - *fast* - command operates in constant or log(N) time.  Used for latency monitoring.
-  - *movablekeys* - keys have no pre-determined position.  You must discover keys yourself.
-
-
-### Movable Keys
+  - `write`: command may result in modifications
+  - `readonly`: command will never modify keys
+  - `denyoom`: reject command if currently out of memory
+  - `admin`: server admin command
+  - `pubsub`: pubsub-related command
+  - `noscript`: deny this command from scripts
+  - `random`: command has random results, dangerous for scripts
+  - `sort_for_script`: if called from script, sort output
+  - `loading`: allow command while database is loading
+  - `stale`: allow command while replica has stale data
+  - `skip_monitor`: do not show this command in MONITOR
+  - `asking`: cluster related - accept even if importing
+  - `fast`: command operates in constant or log(N) time. Used for latency monitoring.
+  - `movablekeys`: keys have no pre-determined position. You must discover keys yourself or use `key-specs` (starting from Redis 7.0).
```

Comment:
we should state that it refers to firstkey,lastkey,step being incomplete, and the client needs to either use COMAND GETKEYS, or keys-specs (>7.0)